app/uk/gov/hmrc/transitmovementsauditing/models/ObjectSummaryWithFields.scala

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.transitmovementsauditing.models
+
+import uk.gov.hmrc.objectstore.client.ObjectSummaryWithMd5
+
+case class ObjectSummaryWithFields(objectSummary: ObjectSummaryWithMd5, fields: String)
```

Comment:
The fields should be a `JsObject` that contains key value pairs, shouldn't it? The whole point of extracting these fields is to ensure that CIP and query things, we need to make it queryable and we do that by sending Json. 

With that in mind, shouldn't the fields be extracted to key value pairs, rather than strings?

Reply:
Agree - will make queryable.